ts/src/pro/bitfinex2.ts

```diff
@@ -993,7 +993,7 @@ export default class bitfinex2 extends bitfinex2Rest {
             'ACTIVE': 'open',
             'CANCELED': 'canceled',
             'EXECUTED': 'closed',
-            'PARTIALLY FILLED': 'open',
+            'PARTIALLY': 'open',
```

Comment:
@pcriadoperez This is only working because we break the original order status by space and take the first element, which is a bit hacky, but ok, we can improve it later. 
![image](https://github.com/ccxt/ccxt/assets/43336371/f75219f8-fcff-4913-876b-0e79998c2d54)